codecs/codec-framework/pom.xml

```diff
@@ -17,32 +17,34 @@
   <dependencies>
 
     <dependency>
-      <groupId>org.immutables</groupId>
-      <artifactId>value</artifactId>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <scope>test</scope>
+      <exclusions>
```

Comment:
I think we can put this exclusion in the parent pom definition.

Reply:
Moving this.